pyke/lightcurve.py

```diff
@@ -218,8 +200,94 @@ def cdpp(self, transit_duration=13, savgol_window=101, savgol_polyorder=2,
         cdpp_ppm = np.std(mean) * 1e6
         return cdpp_ppm
 
-    def to_csv(self):
-        raise NotImplementedError()
+
+    def periodogram(self, minper=None, maxper=None, nterms=1):
+        """
+        Creates a periodogram object
```

Comment:
not sure if that's the best purpose for a method.
maybe we want to implement something more specific?